docs/source/userguide/command_line.rst

```diff
@@ -152,7 +180,12 @@ The command above would list all GeoIPS Plugin Packages with information includi
 package name, docstring, and current version number. For a listing of what columns you
 can filter by, run ``geoips list <cmd_name> --columns help``.
 
+.. _geoips_list_interface:
+
+:ref:`geoips list interface <geoips_list_interface>`
```

Comment:
This has changed now, though, right?

Reply:
No it has. I still think it might be worth just waiting to update the documentation until all CLI PRs are merged in. Up to you though.